src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './calculator.css';
+/* eslint-disable */
+class Calculator extends React.Component {
+  render() {
+    return (
+
+      <div className="calculator">
+        <input type="text" className="calculator-screen" value="0" disabled />
+        <div className="calculator-keys">
+
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+
+          <button type="button" className="operator">÷</button>
+
+          <button type="button">7</button>
+          <button type="button">8</button>
+          <button type="button">9</button>
+
+          <button className="operator">x</button>
+
+          <button type="button">4</button>
+          <button type="button">5</button>
+          <button type="button">6</button>
+
+          <button className="operator">-</button>
+
+          <button type="button">1</button>
+          <button type="button">2</button>
+          <button type="button">3</button>
+
+          <button type="button" className="operator">+</button>
+
+          <button type="button" className="zero">0</button>
+
+          <button type="button">.</button>
+
+          <button type="button" className="operator">=</button>
+        </div>
+      </div>
+    );
+  }
+}
+
```

Comment:
- [ ] The `calculator.js` component should match the following given design. Please make it look like the design. Please make same color on the buttons and screen as well.
### Your design
![his](https://user-images.githubusercontent.com/81727988/220923656-38da7480-8734-4eaf-ac62-a2c4d98abb81.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/220923736-5eee7e72-f184-4050-8686-6c4b69ebea6b.png)